src/screens/Calendar.tsx

```diff
@@ -1,22 +1,45 @@
-import React from 'react'
-import { StatusBar, StyleSheet, Text, View } from 'react-native'
-import { Agenda } from 'react-native-calendars'
+import React, {useState} from 'react'
+import { StatusBar, StyleSheet, Text, View, ScrollView } from 'react-native'
+import { Agenda, CalendarProvider } from 'react-native-calendars'
 
 export default function Calendar(): JSX.Element {
+    var date = new Date();
```

Comment:
We shouldn't have semicolons in our code.
```suggestion
    var date = new Date()
```